### PR TITLE
Add servicemonitor to federate metrics from cluser-monitor prometheus

### PR DESCRIPTION
**Pull request by 4n4nd**
This will scrape the cluster monitoring prometheus every 60 seconds

### CODE REVIEW

File: odh/overlays/moc/monitoring/servicemonitors/cluster-metrics-servicemonitor.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+kind: ServiceMonitor
+metadata:
+  name: cluster-monitor-federation
+  labels:
+    monitor-component: cluster-resources
+spec:
+  endpoints:
+    - bearerTokenSecret:
+        key: token
+        name: prometheus-k8s-token-g9n8s
```
Comment (tumido) [inline odh/overlays/moc/monitoring/servicemonitors/cluster-metrics-servicemonitor.yaml:12]: This token secret is unique for each SA "instance" (hence the `-g8n8s` suffix). If we would redeploy the Service Account, this token secret name would change. You'd probably need to create a new SA secret with a static name and refer to that one.

https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#to-create-additional-api-tokens

```
apiVersion: v1
kind: Secret
metadata:
  name: a-new-static-name
  annotations:
    kubernetes.io/service-account.name: prometheus-k8s
type: kubernetes.io/service-account-token
```

**Review by tumido — CHANGES_REQUESTED** [review]

File: odh/overlays/moc/monitoring/servicemonitors/cluster-metrics-servicemonitor.yaml (hunk 1 above)
Comment (4n4nd) [reply odh/overlays/moc/monitoring/servicemonitors/cluster-metrics-servicemonitor.yaml]: @tumido I have added a secret in the secrets dir can you give it a look?

**Review by tumido — COMMENTED** [review]
@4n4nd I think that should do the trick :+1: 

/lgtm one more pair of eyes please

**Pull request merged**